robot_skills/src/robot_skills/perception.py

```diff
@@ -296,31 +307,22 @@ def get_rgb_depth_caminfo(self, timeout=5):
         :param timeout: How long to wait until the images are all collected.
         :return: tuple(rgb, depth, depth_info) or a None if no images could be gathered.
         """
-        event = Event()
-
-        def callback(rgb, depth, depth_info):
-            rospy.loginfo('Received rgb, depth, cam_info')
-            self._image_data = (rgb, depth, depth_info)
-            event.set()
-
-        # camera topics
-        depth_info_sub = message_filters.Subscriber('{}/depth_registered/camera_info'.format(self._camera_base_ns),
-                                                    CameraInfo)
-        depth_sub = message_filters.Subscriber('{}/depth_registered/image'.format(self._camera_base_ns), Image)
-        rgb_sub = message_filters.Subscriber('{}/rgb/image_raw'.format(self._camera_base_ns), Image)
-
-        ts = message_filters.ApproximateTimeSynchronizer([rgb_sub, depth_sub, depth_info_sub],
-                                                         queue_size=1,
-                                                         slop=10)
-        ts.registerCallback(callback)
-        event.wait(timeout)
```

Comment:
We need to decide how we want to manage these subscribers

Reply:
Discuss with @ar13pit

Reply:
Done, we are restoring the old implementation. After which we work on RGBD python stuff.